spec/requests/admin/admin_holidays_controller_spec.rb

```diff
@@ -65,5 +65,102 @@ module Admin::DiscourseCalendar
         end
       end
     end
+
+    describe "#disable" do
+      context "when the calendar plugin is enabled" do
+        let(:calendar_enabled) { true }
+        let(:dia_de_la_independencia) { { holiday_name: "Día de la Independencia", region_code: "mx" } }
+
+        context "when an admin is signed in" do
+          before do
+            sign_in(admin)
+          end
+
+          it "disables a specific holiday in a specific region" do
+            post "/admin/discourse-calendar/holidays/disable.json",
+              params: {
+                disabled_holiday: dia_de_la_independencia
+              }
+
```

Comment:
```suggestion
disabled_holiday = DiscourseCalendar::DisabledHoliday.last
```

Reply:
Nice suggestion, I made this change! :+1: